iina/VideoView.swift

```diff
@@ -444,12 +444,12 @@ extension VideoView {
     guard let mpv = player.mpv else { return false }
 
     guard let primaries = mpv.getString(MPVProperty.videoParamsPrimaries), let gamma = mpv.getString(MPVProperty.videoParamsGamma) else {
-      Logger.log("HDR primaries and gamma not available", level: .debug, subsystem: hdrSubsystem)
+      Logger.log("HDR primaries and gamma not available", level: .warning, subsystem: hdrSubsystem)
```

Comment:
May need to rethink using warning for this one. If I remember correctly I think it is normal for the primaries and gamma to not initially be available.